manim/utils/config_utils.py

```diff
@@ -29,9 +30,11 @@ def _parse_file_writer_config(config_parser, args):
 
     # Handle input files and scenes.  Note these cannot be set from
     # the .cfg files, only from CLI arguments
-    fw_config["input_file"] = args.file
-    fw_config["scene_names"] = args.scene_names if args.scene_names is not None else []
-    fw_config["output_file"] = args.output_file
+    # Don't set these if the a subcommand is invoked
```

Comment:
Issue: this comment is just repeating what the next line of code is saying. I suggest changing to "If a subcommand is given, manim will not render a video and thus input/output files are not needed".

Reply:
Tip: don't say what the code is doing, say why it is doing it.

Reply:
Thanks for the suggestion! I'll be more descriptive when writing comments from now on :)